sycl/plugins/unified_runtime/ur/adapters/hip/kernel.hpp

```diff
@@ -55,6 +56,7 @@ struct ur_kernel_handle_t_ {
     args_size_t ParamSizes;
     args_index_t Indices;
     args_size_t OffsetPerIndex;
+    std::set<const void *> PtrArgVals;
```

Comment:
[NIT] feel free to ignore that, if I understand correctly, I feel we could rename that to `PtrArgs` only  as `Vals` word could make some confusion that it points to the values stored inside the Ptrs (after dereference it).

Reply:
Yeah. Makes sense.